Governance.md

```diff
@@ -0,0 +1,323 @@
+
+If a Council member becomes inactive in the project for a period of one year,
+they will be considered for removal from the Council. Before removal, they will
+be contacted to determine if they plan on returning to active participation. If
+not, they will be removed immediately upon a vote of the remaining Council
+members, and their vacancy will be filled in the process outlined above.
+
+The Council reserves the right to eject current Members if they are deemed to be
+actively harmful to The Project's well-being, and attempts at communication and
+conflict resolution have failed. This requires the consensus of the remaining
+Members.
+
+Current and retired Steering Council Members will be listed on
+[Steering Council Membership document](./membership/SteeringCouncil.md).
+
+To the maximum extent possible, Council discussions and activities will be
+public and done in collaboration and discussion with the Project Contributors
+and Community. The Council will have a private mailing list that will be used
+sparingly and only when a specific matter requires privacy. When private
+communications and decisions are needed, the Council will do its best to
+summarize those to the Community after eliding personal/private/sensitive
+information that should not be posted to the public internet.
+
+#### Conflict of Interest
+
+It is expected that the Council Members will be employed at a wide range of
+companies, universities, and non-profit organizations. Because of this, it is
+possible that Members will have conflicts of interest. Such conflict of
+interests include, but are not limited to:
+
+-   financial interests, such as investments, employment, or contracting work,
+    outside of The Project that may influence their work on The Project
+-   access to proprietary information of their employer that could potentially
+    leak into their work with The Project
+
+All members of the Council shall disclose to the rest of the Council any
+conflict of interest they may have. Members with a conflict of interest in a
+particular issue may participate in Council discussions on that issue but must
+recuse themselves from voting on the issue.
+
+#### Subcommittees
+
+The Council can create subcommittees that provide leadership and guidance for
+specific aspects of the project. Subcommittees may include non-Council members
+as appropriate. Like the Council as a whole, subcommittees should conduct their
+business in an open and public manner unless privacy is specifically called for.
+Private subcommittee communications should happen on the main private mailing
+list of the Council unless specifically called for.
+
+The Project shall have a Code of Conduct Subcommittee responsible for
+maintaining the [GeoPandas Code of Conduct](CodeOfConduct.md), responding to
+reports of Code of Conduct Violations, and enforcing Code of Conduct Violations.
+This subcommittee shall not be entirely composed of Steering Council Members.
```

Comment:
Indeed, I think we should try to minimize overlap in practice. I know this can be hard in volunteer organizations, but the load of a CoC member should (hopefully!) be much lower than that for a Council member, so it may be easier to solicit them from the community.